internal/cluster/cluster.go

```diff
@@ -921,6 +924,50 @@ func (m *Manager) PrepareForInstallation(ctx context.Context, c *common.Cluster,
 	return err
 }
 
+func (m *Manager) UploadEvents() {
+	if !m.leaderElector.IsLeader() {
+		m.log.Infof("Not a leader, exiting upload")
+		return
+	}
+
+	if !m.uploadClient.IsEnabled() {
+		m.log.Infof("Event uploading is not enabled")
+		return
+	}
+	var (
+		requestID = requestid.NewID()
+		ctx       = requestid.ToContext(context.Background(), requestID)
+		log       = requestid.RequestIDLogger(m.log, requestID)
+		err       error
+	)
+
+	clustersToUpload, err := common.GetClustersFromDBWhere(m.db, common.SkipEagerLoading, common.SkipDeletedRecords,
+		"uploaded = ? AND (status in (?) OR (status = ? AND install_completed_at > ?))",
+		false, []string{models.ClusterStatusInstalled, models.ClusterStatusCancelled, models.ClusterStatusError},
+		models.ClusterStatusAddingHosts, strfmt.NewDateTime())
+	if err != nil {
+		log.WithError(err).Warnf("failed to get cluster upload table")
```

Comment:
nit: the error message can be rephrased (there is no longer cluster-upload table in this version)

Reply:
Good catch, thank you!